sdks/explanations/data-model/data-model-patient.md

```diff
@@ -0,0 +1,71 @@
+---
+slug: patient
+tags:
+    - data model
+    - data owner
+    - patient
+---
+# Patient
+
+A Patient is the subject of all the medical processes. They can be invited by Healthcare Professionals, who can also 
+associate Data Samples and Healthcare Elements to them.
+As a [Data Owner](/sdks/glossary#data-owner), a Patient can ask to access those data as well as creating Data Samples and
+Healthcare Elements by themselves.
+
+## When to Use a Patient?
+
+You should use a Patient when you need to represent the recipient of the medical care in your application.
+
+## How a Patient is Related to Other Entities?
+
+A Healthcare Professional can create a Patient.  
+A Patient can create Data Samples and Healthcare Elements.  
+A Patient can ask to access a Data Sample or a Healthcare Element by creating a Notification to a Healthcare Professional.  
+
+## Examples
+
+### A Patient Registering Their Symptoms
+
+A Patient wants to add some symptoms she is experiencing (headache) by adding a new Data Sample.
+Then, she adds the information that her period started as a Healthcare Element associated to the Data Sample.
```

Comment:
The example shows stuff the other way around: first create the healthcare element then create the data sample:
```suggestion
A Patient has a period tracking app and wants to register that her period started and that she is experiencing a headache.
She will create a Healthcare Element to register that her period started, then she will create a Data Sample associated to the Healthcare Element to register the headache.
```